couchpotato/core/media/movie/providers/torrent/iptorrents.py

```diff
@@ -11,9 +11,9 @@ class IPTorrents(MovieProvider, Base):
 
     cat_ids = [
         ([87], ['3d']),
-        ([89], ['bd50']),
-        ([48], ['720p', '1080p']),
-        ([101], ['2160p']),
+        ([89, 90], ['bd50']),
```

Comment:
Why touching these? Does it have something to do with the login problem?

Reply:
with out this change movies with in the IPT Quality   

- Movie/BD-Rip
- Movie/MP4
- Movie/Web-DL
- Movie/x254

will never be found with the search, I had it with a certain movie during testing

Reply:
I get it for 90 and 20, but not for.
100 is not just a 2160p.
100 = x265 and it has movies with 720p and 1080p qualities.

Reply:
I get it for 90 and 20, but not for.
100 is not just a 2160p.
100 = x265 and it has movies with 720p and 1080p qualities.

Reply:
Agreed, though this was the best mapping I could come up with, not sure what the perfect solution is here?